### PR TITLE
Feature/show2 1343 complete profile

### PR DESCRIPTION
**Pull request by alantoa**
# Why

- [Complete Profile](https://www.notion.so/showtime-xyz/Sign-Up-Complete-Profile-8274c1cd57be4d05803bdfd02e72be7b) 
-  [Replace “Expired” with “Time out” ](https://showtime-rq88331.slack.com/archives/C02PXGK3V8D/p1667492132005229)

# How

- improve the edit profile design. 
- if the user hasn't completed the profile, show the complete profile screen when the open app every time. 
- update the icon in the system. 
- fixed the `ModalSheet` component which will resemble `ModalScreen` on the web. 

# Test Plan

use your new wallet to sign in and check if the complete profile flow looks good! 

https://user-images.githubusercontent.com/37520667/200216557-62ba4c08-e741-4932-8b10-b0c3d29f786e.MOV

# Todo 

- [ ] Connect Twitter and Facebook, need to wait for the API key, will complete it in the next PR!

### CODE REVIEW

**Comment by intergalacticspacehighway** [comment]
> Connect Twitter and Facebook, need to wait for the API key, will complete it in the next PR!

Twitter connect method is on this [branch](https://github.com/showtime-xyz/showtime-frontend/tree/feat/twitter-login). It has a dependency on this task though. https://showtime-rq88331.slack.com/archives/C02QD3J7DJM/p1667387160985879

I can pick it with social logins!

**Review by intergalacticspacehighway — APPROVED** [review]
LGTM 🔥

**Comment by alantoa** [comment]
@intergalacticspacehighway yeah, I know social logins are related to this, so I haven't merged your branch.
I have added some icons for the `design-system`, in order to avoid file conflicts, I'll push the code to your branch when I merge this branch. 😏

File: packages/app/components/drop/drop-form.tsx [hunk 1]
```diff
@@ -169,17 +167,12 @@ export const DropForm = () => {
...
-    !userProfile?.data.profile.img_url
-  ) {
+  if (user.isIncompletedProfile) {
     return (
       <CompleteProfileModalContent
-        title="Tell your collectors more about yourself"
-        description="Complete your profile first to create this drop. It will take around 1 minute."
-        cta="Complete profile to drop"
+        title="Just one more step"
+        description="You need complete your profile to create/claim drops. It only takes about 1 min"
```
Comment (axeldelafosse) [inline packages/app/components/drop/drop-form.tsx]: "create/claim" -> "create"

File: packages/app/components/claim/claim-form.tsx [hunk 2]
```diff
@@ -139,20 +138,12 @@ export const ClaimForm = ({
...
+  if (isIncompletedProfile) {
     return (
       <CompleteProfileModalContent
-        title={`Show ${getProfileName(
-          creatorProfile?.data?.profile
-        )} who you are!`}
-        description="Complete your profile first to collect this drop. It will take around 1 minute."
-        cta="Complete profile to collect"
+        title="Just one more step"
+        description="You need complete your profile to create/claim drops. It only takes about 1 min"
```
Comment (axeldelafosse) [inline packages/app/components/claim/claim-form.tsx]: "create/claim" -> "collect"

**Review by axeldelafosse — APPROVED** [review]
LFG ✨

File: packages/app/components/claim/claim-form.tsx (hunk 2 above)
Comment (alantoa) [reply packages/app/components/claim/claim-form.tsx]: thank you!

**Pull request merged**